.github/workflows/codeql.yaml

```diff
@@ -0,0 +1,54 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches:
+    - "main"
+  pull_request:
+    branches:
+    - "main"
+  schedule:
+  - cron: '16 11 * * 5'
+
+permissions:
+  contents: read
+
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: 'ubuntu-22.04'
+    timeout-minutes: 360
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language:
+        - go
```

Comment:
unnecessary matrix (at least for now), so let's remove

Reply:
Updated